src/api/app/models/cloud/backend/upload_job.rb

```diff
@@ -33,6 +33,15 @@ def self.create(user, params)
         new(exception: exception.message)
       end
 
+      def self.all(user)
+        xml = ::Backend::Api::Cloud.status(user)
```

Comment:
Nitpick: All this logic for talking with the backend could be in the Backend::* lib. We need to think about it, I mean the lib should answer with basic types, not an XML to parse...